### ISSUE TITLE
warning: constant ::Fixnum is deprecated

### ISSUE THREAD

**Issue by MatUrbanski** [AuthorizeNet/sdk-ruby#106]
Gem is incompatible with Ruby 2.4:
```
lib/authorize_net/api/schema.rb:2525: warning: constant ::Fixnum is deprecated
lib/authorize_net/api/schema.rb:2542: warning: constant ::Fixnum is deprecated
lib/authorize_net/api/schema.rb:2544: warning: constant ::Fixnum is deprecated
```

**Comment by jfelchner** [comment]
This is annoying as hell.  What's the time table on fixing this?

**Comment by jfelchner** [comment]
You literally do a find and replace and change `Fixnum` to `Integer`

**Comment by taf2** [comment]
here you go https://github.com/AuthorizeNet/sdk-ruby/pull/121

**Comment by adavidw** [comment]
merged PR #109 to resolve this.
